### PR TITLE
Polymer 3 Conversion

### PR DESCRIPTION
**Pull request by margaree**
The patch changes were to get the `alert-toast` demo working (`openToast` function couldn't be found) and removing extra imports

### CODE REVIEW

File: demo/alert-toast.html [hunk 1]
```diff
@@ -87,18 +87,15 @@ <h3>Default without close button</h3>
 		</div>`;
 
 document.body.appendChild($_documentContainer.content);
-</script>t>
```
Comment (svanherk) [inline demo/alert-toast.html]: Hmm, I wonder why the modulizer messed this up so badly.  The original file looks fine.  On thing we've been doing in other places is changing from `onclick` inline functions to adding `click` event listeners wrapped in a WebComponentsReady` and `RequestAnimationFrame`: https://github.com/Brightspace/simple-overlay/blob/hybrid/demo/simple-overlay.html#L43

Comment (svanherk) [reply demo/alert-toast.html]: If we set it up like this, I don't think we'd need a patch at all.

Comment (margaree) [reply demo/alert-toast.html]: Okay awesome, I'll give that a try

**Comment by margaree** [comment]
I'm going to just rerun the conversion script and open a new PR after I merge https://github.com/BrightspaceUI/alert/pull/22. It looks like it should not longer require a patch.

**Pull request closed without merging**